test/worker/worker.test.ts

```diff
@@ -118,7 +118,7 @@ test("Should not populate Environment with extra_security_groups when empty stri
 });
 
 function expectLambdaEnvironmentContainsNoExtraSecurityGroups(stack: cdk.Stack) {
-    expectCDK(stack).to(haveResource("AWS::Lambda::Function", {
+    expectCDK(stack).to(haveResourceLike("AWS::Lambda::Function", {
```

Comment:
Did this not work before? Or was it affected by the version upgrade?

Reply:
It's related to passing the generated accumulo password in the environment.

Reply:
Reverted back.